README.md

```diff
@@ -135,10 +167,17 @@ For PD, add content below to spec.pd (replace ${CLUSTR_NAME}),
 ```
 
 ### PD scheduler configuration
+
 Increase the PD leader-schedule limit after the cluster is deployed, through sql:
-```
+
+```SQL
 set config pd `leader-schedule-limit`=100;
 ```
 
 ## Limitation
-Current version is only for cluster deployed by TiDB Operator and TLS is eanbled.
\ No newline at end of file
+
+Current version is only for cluster deployed by TiDB Operator and TLS is eanbled.
```

Comment:
`and TLS is eanbled` should be removed

Reply:
addressed in [f31451a](https://github.com/PingCAP-QE/tidb-google-maintenance/pull/4/commits/f31451a5c678639b5e2675a484dfa26cf3832de7)